contrib/ansible/roles/etcd/tasks/main.yml

```diff
@@ -8,6 +8,11 @@
     - restart etcd
   when: not is_atomic
 
+- name: Clean etcd data directory
+  shell: rm -rf {{ etcd_data_dir }}/*
+  notify:
+    - restart etcd
```

Comment:
@yichengq thx, I guess this is the route we'll need to take and make sure the change etcd changes are rolling.